src/icons/states/LocationErrorState.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { boxBlue, white } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    opacity?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const LocationErrorState = ({
+    className = '',
+    color = boxBlue,
+    height = 111,
+    opacity = 0.1,
+    title,
+    width = 130,
+}: Props) => (
+    <AccessibleSVG
+        className={`LocationErrorState ${className}`}
```

Comment:
For BUIE, you would prefix the class with `bdl`. So in this case `bdl-LocationErrorState`

Reply:
I thought `bdl` namespaces applies when we have class that has Box specific style. Is that not the case?

Reply:
nvm We want to make sure this does not have conflict with other styles that the user might have. Curious, any reason we don't want to use classnames?

Reply:
you can use classnames if you want. Though with just a single appending its not as useful

Reply:
Can get rid of the extra space :) but not going to do it for this commit.

Reply:
Going to add classnames since I am addressing comments.